src/primitives.jl

```diff
@@ -210,8 +210,9 @@ dropnull!(X::NullableVector) = deleteat!(X, find(X.isnull)).values # -> Vector
 
 Returns whether or not any entries of `X` are null.
 """
-anynull(X::Any) = any(_isnull, X)           # -> Bool
-anynull(X::NullableArray) = any(X.isnull)   # -> Bool
+anynull(X::Any) = any(_isnull, X)                                           # -> Bool
+anynull{T}(X::AbstractArray{T}) = T <: Nullable ? any(_isnull, X) : false   # -> Bool
```

Comment:
You also need to check `T :> Nullable` too, so you could as well keep checks inside the function.